content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -706,11 +699,9 @@ Current reclaim policies are:
 
 * Retain -- manual reclamation
 * Recycle -- basic scrub (`rm -rf /thevolume/*`)
-* Delete -- associated storage asset such as AWS EBS, GCE PD, Azure Disk,
-  or OpenStack Cinder volume is deleted
+* Delete -- associated storage asset such as AWS EBS or GCE PD volume is deleted
 
-Currently, only NFS and HostPath support recycling. AWS EBS, GCE PD, Azure Disk,
-and Cinder volumes support deletion.
+Currently, only NFS and HostPath support recycling. AWS EBS and GCE PD volumes support deletion.
```

Comment:
Remove "AWS EBS and GCE PD volumes support deletion".

Reply:
According to https://github.com/kubernetes/website/pull/42302#discussion_r1282723644, GCE PD is still in Kubernetes 1.27

Rewording here would be good.
```suggestion
For Kubernetes {{< skew currentVersion >}}, only `nfs` and hostPath` volume types support recycling.
The GCE PD volume type supports deletion only.
```
(and then amend further for _dev-1.28_ once this change merges into that branch)